packages/cli-helpers/src/auth/setupHelpers.ts

```diff
@@ -110,13 +109,17 @@ export const standardAuthHandler = async ({
       addApiPackages(apiPackages),
       installPackages,
       extraTask,
-      notes ? printNotes(notes) : null,
+      notes && {
+        title: 'One more thing...',
+        task: () => {},
```

Comment:
So annoying we can't just put the `console.log` in the task :/ 

I'd add a comment here, explaining why we're doing it the way we are (since it's different to other tasks)

```suggestion
        task: () => {
          // Can't console.log the notes here because of
          // https://github.com/cenk1cenk2/listr2/issues/296
          // So we do it after the tasks have all finished instead
        },
```